frontend/src/dataset_explorer/TableChartDataTable.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import {useTable, usePagination, useSortBy, Column} from "react-table";
+import { Cell, HeaderGroup, Row } from "react-table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import TableFooter from "@material-ui/core/TableFooter";
+import TablePagination from "@material-ui/core/TablePagination";
+import TableSortLabel from "@material-ui/core/TableSortLabel";
+import Paper from "@material-ui/core/Paper";
+import {MetricConfig} from '../data/MetricConfig';
+import { BreakdownVar, BREAKDOWN_VAR_DISPLAY_NAMES } from "../data/Breakdowns";
+import {Tooltip} from '@material-ui/core';
+import WarningRoundedIcon from "@material-ui/icons/WarningRounded";
+import TableContainer from '@material-ui/core/TableContainer';
+import Table from '@material-ui/core/Table';
+import {MetricId} from '../data/variableProviders';
+
+/** Corollary for TableHeader **/
+function TableHeader({ column }: { column: HeaderGroup<any> }) {
+  return (
+      <TableCell {...column.getHeaderProps(column.getSortByToggleProps())}>
+        {column.render("Header")}
+        <TableSortLabel
+            active={column.isSorted}
+            direction={column.isSortedDesc ? "desc" : "asc"}
+        />
+      </TableCell>
+  );
+}
+
+/** Corollary for TableHeaderGroup **/
+function TableHeaderGroup({ group }: { group: HeaderGroup<any> }) {
+  return (
+      <TableRow {...group.getHeaderGroupProps()}>
+        {group.headers.map((col, index) => (
+            <TableHeader column={col} key={index} />
```

Comment:
I'm inclined to actually just inline TableHeader rather than having its own component